db/migrate/20200730221001_add_trusted_for_forum_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddTrustedForForumToUser < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :trusted_for_forum, :boolean, default: false
```

Comment:
Be aware this migration will update all the users in the table, which can take a lot of time. Maybe it will be a good idea to not add a default.

Reply:
Nice catch :smile: